### PR TITLE
incubator-openwhisk-2882 Delete printlns

### PR DESCRIPTION
**Pull request by vooolll**

### CODE REVIEW

**Comment by markusthoemmes** [comment]
Want to give some background why to remove those `println` statements?

**Comment by vooolll** [comment]
@markusthoemmes I want to solve this issue https://github.com/apache/incubator-openwhisk/issues/2882

**Comment by rabbah** [comment]
Thanks for the contribution! And the link to the issue.

Much of the info being leaked is from the dburl property - an alternative would drop or redact some of the sensitive info in this value.

**Comment by vooolll** [comment]
@rabbah Any idea why build failed? I suppose my changes shouldn't affect any tested functionality.

**Comment by rabbah** [comment]
We have a [tool](https://github.com/apache/incubator-openwhisk/blob/master/tools/build/citool) that can help with this:
```bash
> citool monitor 327229861
whisk.core.controller.test.ActivationsApiTests > Activations API should get summary activation by namespace and action name FAILED
```

This is a known heisenbug - I'll kick travis again and look into fixing the bug.

**Comment by vooolll** [comment]
@rabbah ok, I get it

**Comment by cbickel** [comment]
I also think, that only the following 3 variables could contain sensitve information: `db_url`, `target_db` and `source_db`.
But for debugging, when one of these tests fails, the statements are important.

So I'd join @rabbah 's idea of removing the sensitive information form these URLs.

**Comment by markusthoemmes** [comment]
@vooolll ping. What's your status of this?

**Comment by vooolll** [comment]
@cbickel @markusthoemmes deleted only sensitive info from println

File: tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala [hunk 1]
```diff
@@ -468,7 +469,7 @@ class ReplicatorTests
     backupClient.putDoc("testId", JsObject("testKey" -> "testValue".toJson)).futureValue
 
     // Run the replay script
-    val (_, _, replicationId) = runReplay(dbUrl, dbUrl, backupPrefix).head
+    val (_, _, replicationId) = (dbUrl, dbUrl, backupPrefix).head
```
Comment (cbickel) [inline tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala]: Is this change intended?

**Comment by cbickel** [comment]
Thanks a lot for attacking this issue. Removing the credetnials from the test output is very important.

But as mentioned above, I'd suggest to remove only the username and the password, but not the complete URL of the database.

File: tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala (hunk 1 above)
Comment (vooolll) [reply tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala]: no, it is not, will fix it

**Comment by vooolll** [comment]
@cthistle I misunderstood you, ok will change it

**Comment by rabbah** [comment]
@vooolll are you interested in finishing this off?

**Comment by rabbah** [comment]
@cbickel do you have a suggestion for how to do the redaction?

**Comment by vooolll** [comment]
I have problem with running unit tests, I can't find any explanation how to run it, so I wasn't able to finish it

**Comment by vooolll** [comment]
@cbickel I understand it but I can't run this tests, to test is it working correctly

**Comment by rabbah** [comment]
@vooolll did you try `./gradlew :tests:test --tests "your.package.name.TestClass.evenMethodName"`

**Comment by vooolll** [comment]
@rabbah I tried, and I get

`whisk.core.database.test.ReplicatorTests > initializationError FAILED
    pureconfig.error.ConfigReaderException: Cannot convert configuration to a whisk.core.database.CouchDbConfig. Failures are:
      at 'whisk':
        - Key not found: 'couchdb'.
        at pureconfig.package$.getResultOrThrow(package.scala:138)
        at pureconfig.package$.loadConfigOrThrow(package.scala:160)
        at whisk.core.database.test.DatabaseScriptTestUtils$class.$init$(DatabaseScriptTestUtils.scala:40)
        at whisk.core.database.test.ReplicatorTests.<init>(ReplicatorTests.scala:43)
`

**Comment by vooolll** [comment]
@cbickel @rabbah could you please review, I could continue with refactoring, because there is repetitions in database cleanup, or I should do it under another issue?

**Comment by rabbah** [comment]
@vooolll do you have an Apache CLA?

**Comment by dubee** [comment]
This could have used a PG. Looks like upstream dependancies are broken as a result of these changes.
